sources/packages/backend/libs/integrations/src/services/disbursement-schedule/e-cert-generation.service.ts

```diff
@@ -324,8 +324,7 @@ export class ECertGenerationService {
         // Sum all BC grants.
         .reduce((previousValue, currentValue) => {
           return previousValue + +currentValue.effectiveAmount;
```

Comment:
do we need to remove `+` from `+currentValue.effectiveAmount`

Reply:
Removed.